test/cluster/object_store/test_backup.py

```
@@ -280,10 +284,23 @@ async def test_simple_backup_and_restore(manager: ManagerClient, object_storage,
         cf_dir = os.listdir(f'{workdir}/data/{ks}')[0]
 
+        # A backup only ever captures the base table's sstables -- a view has no sstables
+        # of its own worth backing up, since it can always be rebuilt from the base table.
+        # So restoring the base table alone must eventually repopulate the view too.
+        # The view is created only after the snapshot/cf_dir are captured above, since
+        # otherwise it would create a second column-family directory under data/{ks} and
+        # make the os.listdir(...)[0] picks above racy/ambiguous.
+        view = 'test_cf_by_value'
+        if with_view:
+            await cql.run_async(f"CREATE MATERIALIZED VIEW {ks}.{view} AS SELECT * FROM {ks}.{cf} "
+                            "WHERE value IS NOT NULL AND name IS NOT NULL PRIMARY KEY (value, name)")
```

Ruff (0.16.1)

[error] Possible SQL injection vector through string-based query construction

(S608)